runner/browserrunner.ts

```diff
@@ -0,0 +1,211 @@
+/**
```

Comment:
Arrrgh... I wish we could see a diff here. I wonder if two commits would have worked: 1) rename to .ts 2) do the actual changes.